WallsAndHoles/editor.cpp

```diff
@@ -27,6 +28,10 @@ Editor::Editor(QObject *parent)
 
     mToolBar->addAction(mTileMapToolManager->registerTool(QSharedPointer<AbstractTileMapTool>(new TileMapBrushTool(mTileMap)), "Brush Tool"));
 
+    //Height Map Widget
+    QAction *heightMap = mToolBar->addAction("Height Map");
```

Comment:
This 'tool' should be in a tool bar attached to the MapView, and would make more sense to be a toggle. I would suggest making a toolmanager and tool bar in mapView for any view related items. That way if we want multiple views, they could all be a mutually exclusive toggle. And really it doesn't affect anything else, so it doesn't need to be in Editor